src/screens/level.py

```diff
@@ -351,6 +362,18 @@ def start_map_transition(self):
         self.map_transition.activate()
         self.start_transition()
 
+    def decay_health(self):
+        if not self.player.hp <= 10:
```

Comment:
This condition can be simplified and made clearer in the process, it's not immediately obvious what you intend to check.
```suggestion
        if self.player.hp > 10:
```